pkgs/applications/misc/xdragon/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, pkgconfig, gtk3 }:
+
+stdenv.mkDerivation rec {
+  pname = "xdragon";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "mwh";
+    repo = "dragon";
+    rev = "v${version}";
+    sha256 = "sha256-Kq/wiqj9I6W6BnX24Ar7Y4v7PAaXaXEwl80dANL4/zk=";
```

Comment:
I think the convention is to still use base32 hashes:
```
$ nix to-base32 sha256-Kq/wiqj9I6W6BnX24Ar7Y4v7PAaXaXEwl80dANL4/zk=
0fgzz39007fdjwq72scp0qygp2v3zc5f1xkm0sxaa8zxm25g1bra
```

Reply:
you can see the discussions here: https://github.com/NixOS/nixpkgs/pull/89308

essentially, the min-version needs to be at a version which supports it. And I believe it's still at 2.0, but needs to be at 2.2 (could have changed recently)